src/database/sqlite/event/migrations/m056_account_roles.py

```diff
@@ -0,0 +1,48 @@
+from database.sqlite.migration import BaseMigration
+
+
+class Migration(BaseMigration):
+    def forward(self):
+        self.database.execute('ALTER TABLE `account` ADD `fide_id` INTEGER')
+
+        # Create account_role table
+        self.database.execute(
+            'CREATE TABLE `account_role` ('
+            '   `account_id` INTEGER NOT NULL,'
+            '   `tournament_id` INTEGER,'  # NULL for organisers
+            '   `role` TEXT NOT NULL CHECK (`role` IN ("chief_arbiter", "deputy_arbiter", "organiser")),'
```

Comment:
```suggestion
            '   `role` TEXT NOT NULL,'
```
We've been avoiding putting that kind of check in the DB structure, it is really a blockage to evolutions and does not make our data any safer.